src/types/context/app_context.go

```diff
@@ -48,6 +80,23 @@ func (a *AppContext) GetDockerComposeDir() string {
 	return path.Join(a.Location, "docker-compose")
 }
 
+// AddService adds serviceRole to the appConfig object
+func (a *AppContext) AddService(serviceRole string) {
+	if a.Config.Services == nil {
+		a.Config.Services = map[string]types.ServiceSource{}
+	}
+	a.Config.Services[serviceRole] = types.ServiceSource{Location: fmt.Sprintf("./%s", serviceRole)}
+}
```

Comment:
I think this method still belongs on the `AppConfig`. I was thinking we would have the following:
```golang
func (a *AppContext) AddService(serviceRole string) error {
  a.Config.AddService(serviceRole)
  return a.Config.Write(a.Location)
}
```
I liked `Write` being on `AppConfig` as that keeps the read and write of the `AppConfig` in one file.